packages/ndla-ui/src/Article/ArticleByline.tsx

```diff
@@ -151,14 +151,20 @@ export const ArticleByline = ({
   const showPrimaryContributors = suppliers.length > 0 || authors.length > 0;
   const isLearningpath = bylineType === "learningPath";
 
+  const authorLabel: Record<string, string> = {
+    article: "article.authorsLabel",
+    learningPath: "article.authorsLabelLearningpath",
+    external: "article.externalStepAuthorsLabel",
```

Comment:
Sånn for å være konsekvent.
```suggestion
    external: "article.authorsLabelExternal",
```